src/components/Article.jsx

```diff
@@ -62,17 +64,37 @@ if(loading){
             <p id="createdAt"> Created at: {getArticle.created_at}</p>
             </div>
             <h3 id="commentHeader">Comments</h3>
-            <section id="comments">
-                <div id="addCommentsWrapper">
-                    <div id="authorContainer">
-                <img id="authorAvatar" src={props.getUser.avatar_url}></img>
-                 <p id="author"> Username: {props.getUser.username}</p>
+            <section id="addComments">
+                    <div id="UserContainer">
+                        {
+                            props.getUser.username ? <img id="userAvatar" src={props.getUser.avatar_url}></img> : null
+                        }
+                        {
+                            props.getUser.username ?  <p id="username2"> Username: {props.getUser.username}</p> : <p id="noUsername"> Please choose a user to be able to make a comment</p>
+                        }
+               
                     </div>
-                </div>
-                <li id="comments list">
-
-                </li>
+                <label id="commentLable" htmlFor="commentArea">Comment Below</label>
+                <textarea id="commentArea"></textarea>
+               <button className="commentSubmitButton">Submit Comment</button>
```

Comment:
No need for any kind of form right now - this ticket is just about rendering comments!

Having extraneous code like this as part of this PR is a little confusing. For one thing, this is currently a form in name only, since it doesn't have any of the functionality behind it to back it up.

I would recommend keeping the CSS in your back pocket but getting rid of the form elements for now and coming back to them when they're needed later!